src/protocol/mod.rs

```diff
@@ -7,27 +7,82 @@ pub mod sender;
 /// Timestamp in us
 pub type TimeStamp = i32;
 
+/// Duration in us, e.g. RTT
+pub type TimeSpan = i32;
```

Comment:
I'm assuming this isn't a `Duration` because durations aren't signed?

Reply:
Yes, I was intending to turn this into a legit struct with all the right traits, a [Whole Value](http://fit.c2.com/wiki.cgi?WholeValue). Because Duration isn't signed, and Instant subtraction returns Durations, Instant is an unnecessarily cumbersome time API for SRTs use cases. As we evolve TimeStamp to become a first class Whole Value, we should build up a corresponding type (i.e. TimeSpan) to represent an SRT time range/duration/span so we can do time arithmetic in a clear, succinct, and reliable way.

Reply:
Sounds good.

Something we can implement later.